src/ggrc/converters/handlers.py

```diff
@@ -479,6 +480,16 @@ def get_rich_text_value(self):
     return self.raw_value
 
 
+class ConclusionColumnHandler(ColumnHandler):
+
+  """ Handler for design and operationally columns in ControlAssesments """
+
+  def parse_item(self):
+    conclusion_map = {i.lower(): i for i in
```

Comment:
Are we sure about needing this one? If we add conclusions to options table we get this "for free"? Maybe fix `OptionColumnHandler.parse_item` to return `""` instead of `None` but otherwise I don't see usecase for a separate handler?